resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/AMCredentialRenewer.scala

```diff
@@ -59,8 +59,7 @@ private[yarn] class AMCredentialRenewer(
   private var lastCredentialsFileSuffix = 0
 
   private val credentialRenewer =
-    Executors.newSingleThreadScheduledExecutor(
-      ThreadUtils.namedThreadFactory("Credential Refresh Thread"))
+    ThreadUtils.newDaemonSingleThreadScheduledExecutor("Credential Refresh Thread")
```

Comment:
Normally you should avoid making changes that are not related to your PR.

Reply:
I generally agree, but I did this mostly to keep these two classes as congruent as possible. I can change it back if you feel strongly.

Reply:
It's ok since you've done it, I was mostly making a generic comment.